packages/common/src/ui/forms/Panel/PanelField.tsx

```diff
@@ -8,6 +8,7 @@ export const PanelField: React.FC = props => (
     {...props}
     sx={{
       color: theme => theme.palette.form.field,
+      textAlign: 'right',
```

Comment:
I adjust styles slightly to have the rows space-between and the value text align to the right with the field taking up 2/3rds of the space. Mainly this was to suit my need of having longish field names and because I thought it looked decent. Maybe I shouldn't have touched it though

Reply:
looks much better!

Reply:
I think we should update the default `<a>` styles sometime though

Reply:
Sure, what did you have in mind? I just played around with moving the `Link` to have the `PanelField` as the child and that makes it not look like a link, I think

Reply:
didn't have anything in mind, just don't like the default. next time it comes up I'll do something.